### ISSUE TITLE
About Performance Metrics

### ISSUE THREAD

**Issue by wakeupll** [mala-lab/GGAD#3]
Dear Authors,

In _Table2_, you provide AUC and AP.  How to get the final values?
1.End epoch?
2.Cross best evaluate?

Kind regard!

**Comment by fengduqianhe** [comment]
Hi wakeupll， thanks for your question,  because GGAD studies semi-supervised setting with only normal node labeled,  there is no validation set here. After the anomalies are generated it becomes a classification problem,  the final values can be obtained by observing the minimum cls loss function and end epoch in case of overfitting.

**Comment by wakeupll** [comment]
Thank you so much for your response, it helps me a lot!
